server/config/config.go

```diff
@@ -0,0 +1,35 @@
+// Package config contains logic and defaults for the otrego server.
+//
+// Relies on https://github.com/kelseyhightower/envconfig, which is inspired by
+// 12-factor app configuration: https://12factor.net/confi.
+package config
+
+import (
+	"log"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+const (
+	// envVarPrefix is a prefix to all environment variables.  For example, to
+	// specify FOO, you will actually need to specify FOO, unless the struct tag
```

Comment:
I don't understand this "to specify FOO, you will actually need to specify FOO" part. Was it supposed to say "you will actually need to specify OTREGO_FOO"?

Reply:
Ah, yes. Fixed